src/components/AccordionItem/AccordionItem.js

```diff
@@ -121,13 +121,15 @@ export default class AccordionItem extends Component {
         {...other}>
         <Expando
           type="button"
+          aria-expanded={this.state.open}
           className={`${prefix}--accordion__heading`}
-          role="tab"
           onClick={this.handleHeadingClick}>
           <Icon
             className={`${prefix}--accordion__arrow`}
             icon={iconChevronRight}
             description={iconDescription}
+            role={null} // eslint-disable-line jsx-a11y/aria-role
+            aria-hidden={true}
```

Comment:
Could do a standalone `aria-hidden` if we want to use the shorthand form (right?)